frontend/ListAds.elm

```diff
@@ -17,160 +18,231 @@ import Removal
 import State.Config as Config
 import State.ListAds exposing (..)
 import SvgIcons
-import Util exposing (ViewMessage(..), UpdateMessage(..))
+import Util exposing (UpdateMessage(..), ViewMessage(..))
+
 
 type Msg
-  = UpdateAds Int (List Models.Ad.Ad)
-  | FooterAppeared
-  | ChangeDomainFilter (Maybe String)
-  | ChangePositionFilter (Maybe String)
-  | ChangeLocationFilter (Maybe String)
-  | RemovalMessage Removal.Msg
+    = UpdateAds Int (List Models.Ad.Ad)
+    | FooterAppeared
+    | ChangeDomainFilter (Maybe String)
+    | ChangePositionFilter (Maybe String)
+    | ChangeLocationFilter (Maybe String)
+    | ChangeSorting (Maybe String)
```

Comment:
There's always some sort, so this doesn't need to be a `Maybe`. Use an algebraic data type for this instead of stringly typed, see `ListUsers.elm` for an example.